neurons/miners/ethereum/funds_flow_v2/graph_creator.py

```diff
@@ -24,7 +24,7 @@ class Block:
 @dataclass
 class Account:
     address: str
-    balance: Decimal
+    balance: str
```

Comment:
Is there any specific reason for using str instead of Decimal? @StormyTalents

Reply:
We know memgraph have limitation on handling big number python doesn't have.
We should be better to use `string` for those all `wei` rather than using `Decimal`